alpha-admin/src/layout/Dashboard.js

```diff
@@ -102,10 +107,19 @@ export const Dashboard = () => {
               <Typography variant="h6" component="h2" sx={{ ml: 0.5, mt: 0.4 }}>
                 Members (n)
               </Typography>
-
-              <IconButton sx={{ marginLeft: "auto", mr: 0.5 }}>
+              {/* 멤버 관리 모달 (캘린더 관리자에게만 보일 것)*/}
+              <IconButton
+                sx={{ marginLeft: "auto", mr: 0.5 }}
+                onClick={editMemModalOpen}
+              >
+                <EditIcon />
+              </IconButton>
+              <EditMemModal open={editMemOpen} close={editMemModalClose} />
+              {/* 멤버 추가 모달 */}
+              <IconButton sx={{ marginX: 1 }} onClick={addMemModalOpen}>
```

Comment:
![image](https://user-images.githubusercontent.com/2508024/155845860-86b767b3-7298-43e2-9792-231bf9c37d06.png)

브라우저 가로폭이 조정되어 Members (n) 의 텍스트가 2줄이 되는 경우, 아이콘이 타원형으로 늘어납니다.
아이콘을 Box로 감싸주셔서 따로 조정해 주시면 좋을 것 같습니다.

```suggestion
              <Box className={classes.buttonBox}>
                <IconButton
                  className={classes.button}
                  variant="contained"
                  onClick={editMemModalOpen}>
                  <EditIcon />
                </IconButton>
                <EditMemModal open={editMemOpen} close={editMemModalClose} />
                {/* 멤버 추가 모달 */}
                <IconButton
                  className={classes.button}
                  variant="contained"
                  onClick={addMemModalOpen}>
                  <AddIcon />
                </IconButton>
                <AddMemModal open={addMemOpen} close={addMemModalClose} />
              </Box>
```